client/android/app/build.gradle

```diff
@@ -149,11 +149,31 @@ android {
 }
 
 dependencies {
-    compile project(':react-native-maps')
+    compile (project(':react-native-maps')){
+        exclude group: "com.google.android.gms"
+    }
+    compile (project(':react-native-fcm')) {
+        exclude group: "com.google.firebase"
+    }
     compile project(':react-native-code-push')
     compile project(':react-native-vector-icons')
     compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
+    compile ("com.google.firebase:firebase-core:11.6.2"){
```

Comment:
any idea what effect the changes in this file will have?

Reply:
we can probably put the app-compat back, but I think that it was causing compilation trouble. I haven't seen anything particularly broken with it from my use.